src/ingest-pipeline/misc/tools/split_and_create.py

```diff
@@ -199,8 +200,38 @@ def populate(row, source_entity, entity_factory, dryrun=False):
         print(row_df)
     else:
         kid_path = Path(entity_factory.get_full_path(uuid))
+
     row_df.to_csv(kid_path / f"{uuid}-metadata.tsv", header=True, sep="\t", index=False)
     extras_path = kid_path / "extras"
+
+    # ToDo read all component metadata files find the matching row for old_data_path and then refactor info
+    #  as main metadata above before creating the new component metadata tsv
+    if components is not None:
+        for component in components:
+            component_df = pd.read_csv(component.get("metadata-file"))
```

Comment:
I think that the component metadata file is still a TSV by this point. You might need `sep="\t"` here.

Reply:
Great catch